lib/galaxy/webapps/galaxy/api/remote_files.py

```diff
@@ -132,20 +132,18 @@ def __create_jstree( self, directory, disable='folders' ):
         jstree_paths = []
         if os.path.exists( directory ):
             for ( dirpath, dirnames, filenames ) in os.walk( directory ):
-
                 for dirname in dirnames:
                     dir_path = os.path.relpath( os.path.join( dirpath, dirname ), directory )
-                    dir_path_hash = hashlib.sha1( dir_path.encode('utf-8') ).hexdigest()
+                    dir_path_hash = hashlib.sha1(dir_path).hexdigest()
                     disabled = True if disable == 'folders' else False
                     jstree_paths.append( jstree.Path( dir_path, dir_path_hash, { 'type': 'folder', 'state': { 'disabled': disabled }, 'li_attr': { 'full_path': dir_path } } ) )
 
                 for filename in filenames:
                     file_path = os.path.relpath( os.path.join( dirpath, filename ), directory )
-                    file_path_hash = hashlib.sha1( file_path.encode('utf-8') ).hexdigest()
+                    file_path_hash = hashlib.sha1(file_path).hexdigest()
```

Comment:
On the face of it, that seems to be the exact same issue I was getting *until* I removed the extra encoding.  My testing configuration is: 

```
[yoplait@malleus:~/work/import] cat ~/work/galaxy/config/galaxy.ini | grep import_dir
library_import_dir = /Users/yoplait/work/import

[yoplait@malleus:~/work/import] ls -l ~/work/import
total 0
drwxr-xr-x  3 yoplait  staff  102 Mar 10 19:23 Ăfolder

[yoplait@malleus:~/work/import] ls -l ~/work/import/Ăfolder 
total 8
-rw-r--r--  1 yoplait  staff  17 Mar 10 19:20 Ăfile.txt
```